pkg/cmd/pr/checkout/checkout.go

```diff
@@ -161,9 +164,16 @@ func cmdsForExistingRemote(remote *context.Remote, pr *api.PullRequest, opts *Ch
 			cmds = append(cmds, []string{"git", "merge", "--ff-only", fmt.Sprintf("refs/remotes/%s", remoteBranch)})
 		}
 	default:
-		cmds = append(cmds, []string{"git", "checkout", "-b", pr.HeadRefName, "--no-track", remoteBranch})
-		cmds = append(cmds, []string{"git", "config", fmt.Sprintf("branch.%s.remote", pr.HeadRefName), remote.Name})
-		cmds = append(cmds, []string{"git", "config", fmt.Sprintf("branch.%s.merge", pr.HeadRefName), "refs/heads/" + pr.HeadRefName})
+		cmds = append(
+			cmds,
+			[]string{"git", "checkout", "-b", pr.HeadRefName, "--no-track", remoteBranch},
+			[]string{"git", "config", fmt.Sprintf("branch.%s.remote", pr.HeadRefName), remote.Name},
+			[]string{"git", "config", fmt.Sprintf("branch.%s.merge", pr.HeadRefName), "refs/heads/" + pr.HeadRefName},
+		)
+	}
+
+	if opts.Local != "" {
+		cmds = append(cmds, []string{"git", "branch", "-m", pr.HeadRefName, opts.Local})
```

Comment:
This looks like the command first creates a branch with the wrong (auto-generated) name and then renames it to the explicitly specified name. Instead of adding a separate `git branch -m` rename invocation, could this flag simply set the name of the branch to the created originally? E.g.:

```go
localBranch := pr.HeadRefName
if opts.Local != "" {
  localBranch = opts.Local
}
cmds = append(cmds, []string{"git", "checkout", "-b", localBranch, "--no-track", remoteBranch})
```